miscoptions.go

```diff
@@ -294,3 +294,25 @@ func (r *RelayMessageOption) SetRelayMessage(p *RelayMessage) error {
 	*r = b
 	return nil
 }
+
+// Get Client Server Message (e.g. Solicit Advertise ...) into this option (when hopcount = 0 of outer RelayMessage)
```

Comment:
Name this the same as the function.

Add commas between the message types you're listing.

Break to newline after 80 characters.

Add a period.